hack/istio/multicluster/start-kind.sh

```diff
@@ -12,9 +12,15 @@
 SCRIPT_DIR="$(cd $(dirname "${BASH_SOURCE[0]}") && pwd)"
 source ${SCRIPT_DIR}/env.sh $*
 
+KIND_NODE_IMAGE=${KIND_NODE_IMAGE}
+
 start_kind() {
+  local image_flag
```

Comment:
best to set this explicitly to "".

This script doesn't have `set -u` but if it ever does, this will cause an error unless you set it to empty string.